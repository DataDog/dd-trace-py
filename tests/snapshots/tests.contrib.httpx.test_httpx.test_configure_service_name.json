[[
  {
    "name": "http.request",
    "service": "test-httpx-service-name",
    "resource": "http.request",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "http",
    "error": 0,
    "meta": {
      "http.method": "GET",
      "http.status_code": "200",
      "http.url": "http://localhost:8001/status/200",
      "runtime-id": "5f9f1e7420d244069ecdd87f9b4c8568"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 8359
    },
    "duration": 4489000,
    "start": 1632172762073642000
  }],
[
  {
    "name": "http.request",
    "service": "test-httpx-service-name",
    "resource": "http.request",
    "trace_id": 1,
    "span_id": 1,
    "parent_id": 0,
    "type": "http",
    "error": 0,
    "meta": {
      "http.method": "GET",
      "http.status_code": "200",
      "http.url": "http://localhost:8001/status/200",
      "runtime-id": "229e411aac83491d8748a37ab7257407"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 8387
    },
    "duration": 3788000,
    "start": 1632172797967181000
  }],
[
  {
    "name": "http.request",
    "service": "test-httpx-service-name",
    "resource": "http.request",
    "trace_id": 2,
    "span_id": 1,
    "parent_id": 0,
    "type": "http",
    "error": 0,
    "meta": {
      "http.method": "GET",
      "http.status_code": "200",
      "http.url": "http://localhost:8001/status/200",
      "runtime-id": "dc16eaf4c5da4492bfe0ccb8371aaf1c"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 8415
    },
    "duration": 4724000,
    "start": 1632172832809596000
  }],
[
  {
    "name": "http.request",
    "service": "test-httpx-service-name",
    "resource": "http.request",
    "trace_id": 3,
    "span_id": 1,
    "parent_id": 0,
    "type": "http",
    "error": 0,
    "meta": {
      "http.method": "GET",
      "http.status_code": "200",
      "http.url": "http://localhost:8001/status/200",
      "runtime-id": "0c193c5bdf384b6f90ed6e271a8a1857"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 8443
    },
    "duration": 6017000,
    "start": 1632172870002771000
  }],
[
  {
    "name": "http.request",
    "service": "test-httpx-service-name",
    "resource": "http.request",
    "trace_id": 4,
    "span_id": 1,
    "parent_id": 0,
    "type": "http",
    "error": 0,
    "meta": {
      "http.method": "GET",
      "http.status_code": "200",
      "http.url": "http://localhost:8001/status/200",
      "runtime-id": "f0e13f1bbcf34d78a8e2ea327a802378"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 8471
    },
    "duration": 3504000,
    "start": 1632172908900180000
  }],
[
  {
    "name": "http.request",
    "service": "test-httpx-service-name",
    "resource": "http.request",
    "trace_id": 5,
    "span_id": 1,
    "parent_id": 0,
    "type": "http",
    "error": 0,
    "meta": {
      "http.method": "GET",
      "http.status_code": "200",
      "http.url": "http://localhost:8001/status/200",
      "runtime-id": "c34cf008330e4d6bbb0909c990c5993b"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 8499
    },
    "duration": 4149000,
    "start": 1632172948231800000
  }],
[
  {
    "name": "http.request",
    "service": "test-httpx-service-name",
    "resource": "http.request",
    "trace_id": 6,
    "span_id": 1,
    "parent_id": 0,
    "type": "http",
    "error": 0,
    "meta": {
      "http.method": "GET",
      "http.status_code": "200",
      "http.url": "http://localhost:8001/status/200",
      "runtime-id": "6330f633b38b446dbcc7ddfaf69f077e"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 9061
    },
    "duration": 5691000,
    "start": 1632173159436216000
  }]]
