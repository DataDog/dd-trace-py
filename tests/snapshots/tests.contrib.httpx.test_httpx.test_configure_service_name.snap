[[{"name" "http.request"
   "service" "test-httpx-service-name"
   "resource" "http.request"
   "type" "http"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1620856200902706000
   "duration" 6542000
   "meta" {"runtime-id" "03c9b32fb3ca4932a59d527e479a69f9"
           "http.method" "GET"
           "http.url" "http://localhost:8001/status/200"
           "http.status_code" "200"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 68931
              "_dd.tracer_kr" 1.0}}]]
